bindings/mlir/qiskit/QiskitTranslation.cpp

```
@@ -0,0 +1,68 @@
+#include <llvm/ADT/StringSet.h>
+
+#include <algorithm>
+#include <cstdint>
+#include <stdexcept>
+#include <string>
+#include <string_view>
+#include <vector>
+
+namespace mqt::bindings::qiskit {
+
+std::uint32_t validateRegisterLayout(const std::vector<Register>& registers,
```

Cppcheck (2.21.0)

[style] The function 'validateRegisterLayout' is never used.

(unusedFunction)